app/services/experiments/table_view_service.rb

```diff
@@ -97,11 +99,18 @@ def id_presenter(my_module)
     end
 
     def due_date_presenter(my_module)
-      if my_module.due_date
-        I18n.l(my_module.due_date, format: :full_date)
-      else
-        ''
-      end
+      {
+        id: my_module.id,
+        data: ActionController::Base.new.render_to_string(
+          partial: 'experiments/table_due_date.html.erb',
+          locals: { my_module: my_module,
```

Comment:
pass to partial only `my_module` and `user`. And call all other helpers inside partial. This will be much more cleaner.